packages/storybook8/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -338,16 +338,31 @@ const Docs: () => JSX.Element = () => {
 
       <div ref={refDisplayAttachments}>
         <Heading>Display Messages with Attachments</Heading>
+        <Subtitle>Basic Usage: Default Attachment Rendering</Subtitle>
         <DetailedBetaBanner />
         <Description>
-          The MessageThread component supports rendering of message attachments, including multiple ways to customize
-          it. Developers can opt to use the default attachment rendering by not providing `attachmentOptions`. In the
-          following example, the default attachment rendering is shown with an attachment on the first chat message. By
-          default, the browser `window.open` method will be called with the target URL.
+          MessageThread component renders message attachment out of box without any additional configuration. Simply
+          provide a list of `ChatMessages` with attachments of type `AttachmentMetadata` and the component will render
+          the message content along with associated attachments.
         </Description>
         <Canvas mdxSource={MessageWithAttachmentText}>
           <MessageWithAttachment />
         </Canvas>
+        <Description>
+          If the identity of message sender is a Microsoft Teams user, the attachment will be rendered with an `open`
+          icon shown below.
+        </Description>
+        <Canvas mdxSource={MessageWithAttachmentText}>
+          <MessageWithAttachmentFromTeams />
+        </Canvas>
+        <Subtitle>Advanced Usage: Customizing Attachment Rendering</Subtitle>
+        <DetailedBetaBanner />
+        <Description>
+          MessageThread component also support multiple ways to customize the rendering. Developers can opt to use the
```

Comment:
Would be nice to have an example of customizing the rendering, rather than mentioning it and then using the default.

Reply:
hi emlyn, example is at line 371